js/api-augment/definitions/msa.ts

```diff
@@ -47,14 +36,24 @@ export default {
       ],
       type: "Option<Vec<SchemaGrantResponse>>",
     },
+    getKeysByMsaId: {
+      description: "Fetch Keys for an MSA Id",
+      params: [
+        {
+          name: "msa_id",
+          type: "MessageSourceId",
+        },
+      ],
+      type: "Option<Vec<KeyInfoResponse>>",
```

Comment:
hmm, thought I left this comment before but I can't seem to find it.
Since `KeyInfoResponse` is only used by this RPC, it seems like it would be more efficient not to duplicate `msa_id` in each array item in the response. So perhaps the definition should be:
```
KeyInfoResponse: {
      keys: "Vec<AccountId>",
      msa_id: "MessageSourceId",
}
```

and then the RPC return value would just be `Option<KeyInfoResponse>` instead of `Option<Vec<KeyInfoResponse>>`

Reply:
applied